emuu-client/src/components/UserProfile/Feeds.js

```diff
@@ -1,41 +1,47 @@
-import React, {useState} from 'react'
-import './Feeds.scss';
-import {Avatar} from '@mui/material';
-
-
-function Feeds(){
-const[UserPostDescrip,setUserPostDescrip]=useState("");
-    return (
-        <div className="Main-Feeds-Container">
-            <div className="top-container">
-                <div className="Feed-input container">
-
-                    <div className="avatar-container">
-                    <Avatar>src="https://wallpaperaccess.com/full/170249.jpg"
-                    </Avatar></div>
-                    <div className="input-container"></div>
-                        <input id="Post-Desc" placeholder="What's on your mind"
-                        value={UserPostDescrip}
-                        onChange={(e)=>{setUserPostDescrip(e.target.value)}}/>
-
+import React, { useEffect, useState } from "react";
+import "./Feeds.scss";
+import { Avatar } from "@mui/material";
+
```

Comment:
Delete this line

Reply:
Empty lines are not supposed to be deleted after the last import state. Used - Prettier...